packages/oo7-substrate/src/bonds.js

```diff
@@ -4,7 +4,7 @@ const { nodeService } = require('./nodeService')
 const { SubscriptionBond } = require('./subscriptionBond')
 const { BlockNumber, Hash } = require('./types');
 const { decode, encode } = require('./codec');
-const { stringToBytes, hexToBytes, bytesToHex } = require('./utils')
+const { stringToBytes, hexToBytes, bytesToHex ,toLE} = require('./utils')
```

Comment:
Incorrect spacing, we're also waiting for the builds to start passing (see #12 )